packages/block-editor/src/components/block-switcher/style.scss

```diff
@@ -26,15 +26,6 @@
 	}
 }
 
-.block-editor-block-switcher__toggle-text {
-	margin-left: $grid-unit-10;
-
-	// Account for double label when show-text-buttons is set.
-	.show-icon-labels & {
-		display: none;
-	}
-}
-
```

Comment:
It looks like there are more custom styles that can be removed, but out of scope for this PR.